Pipfile

```diff
@@ -18,6 +18,9 @@ pytest-cov = "*"
 pydocstyle = "*"
 ipython = "*"
 awscli = "*"
+sphinx = "*"
+sphinx-rtd-theme = "*"
+m2r = "*"
```

Comment:
What is `m2r`?

Reply:
markdown to rst, used to convert markdown to reStructuredText, though some things don't convert 100%